### PR TITLE
Feature/info

### PR DESCRIPTION
**Pull request by LaurAlejandraC**
Retrieve file information

### CODE REVIEW

File: router.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+func notFound(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusNotFound)
+}
+
+func main() {
+	router := mux.NewRouter().StrictSlash(true)
+	router.HandleFunc("/info/{id}", access.GetInformation)
+	router.HandleFunc("/", func(rw http.ResponseWriter, req *http.Request) {
+		fmt.Println("Default")
```
Comment (maasencioh) [inline router.go:18]: Why is this?

File: access/utils.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package access
+
+import(
+	"net/http"
+	"log"
+	"fmt"
+)
+
+const DBurl string = "http://127.0.0.1:5984/blinkbox_files"
```
Comment (maasencioh) [inline access/utils.go:9]: this should be available to change with environmental variables

**Review by maasencioh — APPROVED** [review]
Just few nits, but LGTM

File: router.go (hunk 1 above)
Comment (LaurAlejandraC) [reply router.go:18]: In the other branch I fixed it, was testing and I forgot to correct it.

**Pull request closed without merging**